src/wwwroot/services/auth.js

```diff
@@ -65,12 +65,14 @@
     }
 
     function expandPermissions(profile) {
+      // TODO: Review why the revision numbers are being added into the word templates in this file.
+      var templates = " templates ";
       switch (profile) {
         case "reports": return {
           acceptedUrlsPattern: /^#?\/reports\/?(\?.*)?$|^#?\/reports\/downloads\/?(\?.*)?$/,
           defaultUrl: "/reports"
         };
-        case "templates": return {
+        case templates.trim(): return {
```

Comment:
What do you think about to move the patch up and also rename _templates_ as _TEMPLATES_ to make more clear yet that it is a constant and a patch?

```js
function expandPermissions(profile) {
  // TODO: Review why the revision numbers are being added into the word templates in this file.
  var TEMPLATES = " templates ".trim();
  switch (profile) {
    case "reports": return {
      acceptedUrlsPattern: /^#?\/reports\/?(\?.*)?$|^#?\/reports\/downloads\/?(\?.*)?$/,
      defaultUrl: "/reports"
    };
    case TEMPLATES: return {
      // . . .
```